src/AzureAuth.Test/CommandAadTest.cs

```diff
@@ -444,7 +444,7 @@ public void TestEvaluateOptionsWithoutAliasValidCommandLineOptions()
             subject.TokenFetcherOptions.Should().BeEquivalentTo(expected);
         }
 
-        [Test]
+        [Ignore(reason: "No way to inject authflow yet")]

```

Comment:
Potentially, unless we can still get the test coverage we want on TokenFetcher directly.

I was thinking about this yesterday and I think we actually put too much in the TokenFetcher, and it should not be coupled to the AuthFlowFactory. If TokenFetcher instead took a list of AuthFlows, and only coupled to the AuthFlow executor, it would be easy to test.

Reply:
I'll look into having `TokenFetcher` take a list of `AuthFlow` after this gets merged, but for now I think we can leave this as is.